exampleCourse/questions/demo/annotated/MarkovChainGroupActivity/MarkovChains-Intro/workspace/.ipynb_checkpoints/Markov-Chains-1-checkpoint.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
I'm going to stop commenting on every checkpoint file. The best approach is probably to search for all the `.ipynb_checkpoints` directories introduced in this PR and delete them all. If these are likely to come back, we should probably add `.ipynb_checkpoints` to the `.gitignore`?

Reply:
yes, will do